scripts/dev/dockerfile_generator.py

```diff
@@ -48,12 +61,23 @@ def render(image_name):
 def parse_args():
     parser = argparse.ArgumentParser()
     parser.add_argument("image", help="Type of image for the Dockerfile")
+    parser.add_argument(
+        "--adds",
+        help='Paths to use in the ADD command (defaults to ".")',
+        type=str,
+        default=".",
+    )
+    parser.add_argument(
+        "--script_location",
+        help="Location of the python script to run. (Used only for python_formatting)",
+        default="./scripts/ci/run_black_formatting_check.sh",
+    )
```

Comment:
Can we add a check which makes sure that this is only used with our formatting image? We should immediately fail if it's provided otherwise. There may be a built in with arg parse for this, if not we can just do a simple check I think!